<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remittance</class>
 <widget class="QWidget" name="Remittance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_myAccount">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_myAccount">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>43</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>본인 계좌 번호:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Password">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>73</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀 번호:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Password">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_WithdrawAmount">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>103</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>송금액:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_WithdrawAmount">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_yourAccount">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>133</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>상대 계좌 번호:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_yourAccount">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Withdraw">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>송금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ExitRemittance">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
